Build error: invalid use of incomplete type ‘class QVariant’ in AOSC OS testing-proposed, amd64

Full log: http://fars.ee/w1TZ

Highlight:

```
../src/migrator/lib/migratorfactory.cpp: In lambda function:
../src/migrator/lib/migratorfactory.cpp:64:64: error: invalid use of incomplete type ‘class QVariant’
```

--- a/src/migrator/lib/migratorfactory.cpp
+++ b/src/migrator/lib/migratorfactory.cpp
@@ -11,6 +11,8 @@
 #include <QDir>
 #include <QFileInfo>
 #include <QPluginLoader>
+#include <QVariant>
+#include <QSet>
 #include <fcitx-utils/standardpath.h>
 
 namespace fcitx {

